scripts/install-nix-from-closure.sh

```diff
@@ -1,181 +1,700 @@
 #!/bin/sh
+### v 2.02-1-dev
+###
+###############################
+###
+### Nix installation script
+### Universal shell script for for OS Linux environments
```

Comment:
The version in is ambiguous. Is it the version of this script? Is that relevant to the user? If not, the version can be left out.

Why not just replace this with `Automated Nix installation for Linux based operating systems` .

Universal is quite difficult. Does this work on Linux 1.0? If not, just specify Linux kernel > 3.10 or something similar.

Reply:
If there is be a version, it should be the version of nix itself. However this version is not needed.

Reply:
Yes, it was put because installer did not merged, so I put traditional numbering there 8)